Libplanet.Explorer.UnitTests/Common/Action/NopeAction.cs

```diff
@@ -0,0 +1,17 @@
+using Bencodex.Types;
+using Libplanet.Action;
+
+namespace Libplanet.Explorer.UnitTests.Common.Action
+{
+    public class NopeAction : IAction
```

Comment:
Isn't what you intended *noop* (no-op)?

Reply:
You're right. I intended to word with [Nop](https://en.wikipedia.org/wiki/NOP_(code)) but I had the mistake that treats `Nope`  as one of the word's shapes (e.g., *noop* (no-op), nop). I prefer nop than noop but is there what word you prefer? cc @planetarium/libplanet

Reply:
I personally prefer `NoOpAction` over `NopAction` or `NoopAction` as it sharpens word boundaries.

Reply:
I hope to gather more variable other opinions but sadly it doesn't seem like it. I applied as `NoOpAction` in 88125c8.